src/app/body/milestone-details/milestone-details.component.ts

```diff
@@ -100,11 +106,12 @@ export class MilestoneDetailsComponent implements OnInit {
             this.milestoneDataReady = false;
             this.sprintNumber = this.startService.currentSprintNumber;
             this.getMilestoneDetails();
-            console.log("hit in else");
             this.milestoneService.getTasks(this.orgDomain, this.milestoneId);
             this.milestoneService.taskDataStateObservable.subscribe(()=>{
-                  this.getNumberData();
-                  this.taskDataReady = true;
+              this.tasks = this.milestoneService.taskData;
+              this.getNumberData();
+              this.taskDataReady = true;
+              console.log("check2",this.milestoneService.taskData); //To Test in Production
```

Comment:
remove console

Reply:
Sanjay bhaiya said, its need to test in production.